.github/workflows/ci.yml

```diff
@@ -95,6 +99,7 @@ jobs:
         run: |
           sudo apt update
           sudo apt install libpulse-dev libegl1-mesa libopengl0 gstreamer1.0-gl
```

Comment:
Is still necessary to run these commands with the new `setup-qt-libs` action? Probably not, but just checking.

Reply:
Yes. I need to identify which one is missed but `setup-qt-libs` standalone is not enough. 
I may make PR to `setup-qt-libs` when identify which one is missed.

Reply:
Ok, no problem then. It was just a minor thing.